2015/proposals/akshay.md

```diff
@@ -0,0 +1,107 @@
+# Title
+
+Add taxonomic name resolution to the EcoData Retriever to facilitate data 
+science approaches to ecology.
+
+## Abstract
+
+The EcoData Retriever is an engine which automates the tasks of finding,
+downloading, and cleaning up ecological data files, and then stores them in a
+local database of your choice.The program cleans and munges the datasets and
+formats them  before inserting it into your database.
+
+This project will tackle the problem of resolving scientific names stored 
+inconsistently across different datasets. This would allow the users to combine
+different datasets and do interesting things.
+
+## Technical Details
+
+The main component of this project would be to build a system to resolve a 
+species name by sending it to one of many Taxonomic name resolution services 
+and processing the result to determine the best name. This can be done by 
+incorporating the feature in the retriever directly or using the already 
+existing library [pytaxize](https://github.com/sckott/pytaxize) in the retriever. 
+I would propose to use pytaxize and use that in the retriver as this will 
+ensure modularity and help develop pytaxize in the process so that it becomes a 
+complete port of its R counterpart.This will involve a fair amount of work as 
```

Comment:
I would note that the lead developer pytaxize and it's R counter part is one of the mentors for this project.